src/ctelnet.cpp

```diff
@@ -713,10 +713,10 @@ void cTelnet::processTelnetCommand(const string& command)
 
                     if (mpHost->mDiscordAccessFlags & Host::DiscordServerCanSetPresenceId) {
                         infoMessage = tr("[ INFO ]  - Informing Game Server (via GMCP) of Discord Username and Rich "
-                                                     "Presence Guild, Server is allowed to modify Guild to match Game.");
+                                                     "Presence Server is allowed to modify Server to match Game.");
                     } else {
                         infoMessage = tr("[ INFO ]  - Informing Game Server (via GMCP) of Discord Username and Rich "
-                                                     "Presence Guild, Server is prohibited from modifying Guild.");
+                                                     "Presence Server is prohibited from modifying Server.");
```

Comment:
Likewise:
```
    infoMessage = tr("[ INFO ]  - Informing Game Server (via GMCP) of Discord Username and Rich\n"
                                 "Presence Server in use, the Game Server is prohibited from\n"
                                 "changing the Rich Presence Server.");
```
Actually "*changing the Rich Presence Server*" is not entirely the right words to use for this, but saying "*changing the Rich Presence identification number which is used to select the source on Discord of the set of Icons available to use in the Rich Presence display on Discord and to chose the name of the Game that Discord reports as being **played.***" is too much, even if it does reflect what is actually the case... :frowning_face: